pkg/agent/controller/egress/egress_controller.go

```diff
@@ -153,9 +158,10 @@ func NewEgressController(
 	routeClient route.Interface,
 	nodeName string,
 	nodeTransportInterface string,
-	cluster *memberlist.Cluster,
+	cluster memberlist.Interface,
 	egressInformer crdinformers.EgressInformer,
 	podUpdateSubscriber channel.Subscriber,
+	maxEgressIPsPerNode int,
```

Comment:
I am curious about the behavior if this value is changed and the Agent is restarted? Any impact on existing Egresses if the limit is reduced?

Reply:
Would it be better to use `uint` instead of `int` here ? But it may require type conversion at few occasions.